docs/en_us/internal/testing.rst

```diff
@@ -70,6 +70,12 @@ UI Acceptance Tests
    write end-user acceptance tests directly in Python, using the
    framework to maximize reliability and maintainability.
 
+Internationalization Tests
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Any new text that is added should be internationalized and translated.
```

Comment:
I was hoping to keep the stuff that developers refer to when testing (usually prior to merging to master) all in one place. I'm not opposed to putting it elsewhere as needed also, but hoping to have a single go-to page with everything needed. Make sense? Is this fair?